test/gtest/ucp/test_ucp_tag.cc

```diff
@@ -466,9 +466,24 @@ UCP_INSTANTIATE_TEST_CASE(test_ucp_tag_limits)
 
 class test_ucp_tag_nbx : public test_ucp_tag {
 public:
-    struct completion_value_t {
-        uint32_t value;
-        uint32_t *address;
+    class completion_value_t {
+    public:
+        completion_value_t() : m_address(0), m_value(0)
```

Comment:
Use `nullptr` for pointer.

Reply:
Also, can use default init values.

Reply:
you mean remove initialization and use gcc initialization for primitive types?